criu/crtools.c

```diff
@@ -189,7 +189,8 @@ int main(int argc, char *argv[], char *envp[])
 	 * 2) Transmitting data to the image streamer
 	 * 3) Emitting logs (potentially to a pipe).
 	 */
-	signal(SIGPIPE, SIG_IGN);
+	if (signal(SIGPIPE, SIG_IGN) == SIG_ERR)
+		pr_warn("Failed to set a SIGPIPE signal ignore");
```

Comment:
@ivalery111 could you please add `\n` at the end of the message?

Reply:
Yes, of course. My bad.

Reply:
... : %s\n", strerror(errno)

Reply:
or you can use pr_perror in this case. And I would add "return 1" if it fails.

Reply:
Done, thanks.

Reply:
In contrast with `pr_warn`, a new line character is appended to the end of the message with `pr_perror`.

```shell
$ make lint
...
# Do not append \n to pr_perror or fail
! git --no-pager grep -E '^\s*\<(pr_perror|fail)\>.*\\n"'
criu/crtools.c:		pr_perror("Failed to set a SIGPIPE signal ignore.\n");
make: *** [Makefile:414: lint] Error 1
```

Reply:
I didn't know about `make lint` option. Thank you.